.github/actions/install_dbt_requirements/action.yaml

```diff
@@ -8,16 +8,25 @@ inputs:
 runs:
   using: composite
   steps:
-      - name: Setup python
-        uses: actions/setup-python@v4
+      - name: Install uv
+        uses: astral-sh/setup-uv@v3
         with:
-          python-version: '3.11'
-          cache: pip
+          enable-cache: true
+          cache-dependency-glob: |
+            ${{ inputs.dbt_project_dir }}/pyproject.toml
+            ${{ inputs.dbt_project_dir }}/uv.lock
 
-      - name: Install python requirements
-        working-directory: dbt
+      - name: Install Python
+        uses: actions/setup-python@v5
+        with:
+          python-version-file: ${{ inputs.dbt_project_dir }}/.python-version
```

Comment:
Having this file lets us keep our Python version synchronized with `uv`. If you want to use the Python version associated with the project locally, you just `uv python install` with the virtualenv active.